src/modules/users/edit-user/edit-user.html

```diff
@@ -56,7 +56,7 @@
             <div class="form-group">
                 <label class="col-sm-2 col-md-1 control-label cp-padding" for="password">Confirm Password</label>
                 <div class="col-sm-8 col-md-6 inner-addon">
-                    <input type="{{ editUserCntrl.confirmPassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="confirmPassword" ng-model="editUserCntrl.user.confirmPassword" minlength="8">
+                    <input type="{{ editUserCntrl.confirmPassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="confirmPassword" ng-model="editUserCntrl.user.confirmPassword" minlength="9" ng-maxlength="128">
```

Comment:
if you have used ng-maxlength then why you have not used ng-minlength?

Reply:
Same as above comment.